src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java

```diff
@@ -0,0 +1,14 @@
+package com.wcc.platform.domain.cms.pages;
+
+import com.wcc.platform.domain.cms.attributes.Contact;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+
+// TODO: Get photo for About Us hero section.
+// TODO: Use Page or HeroSection record?
+// TODO: Add id to the below record declaration?
```

Comment:
Please add as string for now

Reply:
As agreed during the meeting, we will add ids to all pages in another PR.